### PR TITLE
Sign up email i18n

### PR DESCRIPTION
**Pull request by lin-d-hop**
There are quite a few menial commits in this, as I deployed to staging in order to test emails.

These changes provide i18n for the enterprise welcome email, including the ability to point to our own user forums. User forum is set in the instance content configuration.

An email address in the user signup confirmation email has been updated to read from instance configuration (as opposed to hard coding) as well.

@daniellemoorhead would you be able to schedule this? Thanks!

### CODE REVIEW

**Comment by daniellemoorhead** [comment]
Eek, sorry for the delay @lin-d-hop. Have assigned @mkllnk to review and merge. Cheers!

File: app/views/enterprise_mailer/welcome.html.haml [hunk 1]
```diff
@@ -13,7 +13,7 @@
   = t :email_admin_html, link: link_to('Admin Panel', spree.admin_url)
 
 %p
-  = t :email_community_html, link: link_to('Join the community.', 'http://community.openfoodnetwork.org/')
+  = t :email_community_html, link: link_to(t(:join_the_community), "#{ ContentConfig.community_forum_url }")
```
Comment (mkllnk) [inline app/views/enterprise_mailer/welcome.html.haml]: Is there a reason why you put the url in a string `"#{...}"`?

**Comment by mkllnk** [comment]
Looks good to me. I hope we can push it to production on Friday.

**Comment by sstead** [comment]
@mkllnk All looks to be working correctly...
- The contact email in signup emails is customisable in config > content
- The Forum link is customisable, via new field in Config > content
- If the forum link is left blank, the link returns the user to the homepage
- The main forum link (NOTICES in Aus) is not effected by the config settings.

image

**Pull request merged**